documentation/source/user_section/tutorials/registration_to_template.rst

```diff
@@ -0,0 +1,375 @@
+
+Command: ``sct_deepseg_sc``
+===========================
+
+.. code:: sh
+
+   sct_deepseg_sc -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
+
+   # Input arguments:
+   #   - i: Input image
+   #   - c: Contrast of the input image
+   #   - qc: Directory for Quality Control reporting. QC reports allow us to evaluate the segmentation slice-by-slice
+
+   # Output files/folders:
+   #   - t2_seg.nii.gz: 3D binary mask of the segmented spinal cord
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_propseg_before_after.png
+   :align: center
+   :figwidth: 50%
+
+   Input/output images for ``sct_deepseg_sc``.
+
+.. _vert-labeling-section:
+
+Step 2: Vertebral/disc labeling
+*******************************
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/registration-pipeline-2.png
+   :align: center
+   :figwidth: 75%
+
+   Steps performed by ``sct_label_vertebrae`` and ``sct_label_utils``.
+
+Theory
+======
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/instrumentation-missing-discs.png
+   :align: right
+   :figwidth: 25%
+
+   ``sct_label_vertebrae`` is able to label vertebral levels despite missing discs due to instrumentation.
+
+Next, the segmented spinal cord must be labeled to provide reference markers for matching the PAM50 template to subject's MRI.  The vertebral/disc labeling algorithm works as follows:
+
+  #. The spinal cord is straightened to make it easier to process.
+  #. Then, labeling is done using an automatic method that finds the C2-C3 disc, then finds neighbouring discs using a similarity measure with the PAM50 template at each specific level.
+
+     - The C2-C3 disc is used as a starting point because it is a distinct disc that is easy to detect (compared to, say, the T7-T9 discs, which are indistinct compared to one another).
+     - The labeling algorithm uses several priors from the template, including the probabilistic distance between adjacent discs and the size of the vertebral discs. These priors allow it to be robust enough to handle cases where instrumentation results in missing discs.
+
+  #. Finally, the spinal cord and the labeled segmentation are both un-straightened.
+
+
+Label files are produced for both vertebral levels and intervertebral discs, and either can be used for the later registration steps. For vertebral levels, the convention is to place labels as though the vertebrae were projected onto the spinal cord, centered in the middle of the vertebral level. For discs, the convention is to place labels on the posterior tip of the disc.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/vertebral-labeling-conventions.png
```

Comment:
that's a SUPER IMPORTANT image, which will will be referring to often, so i suggest:
- making it bigger
- associating it with a specific sub-heading so we can refer to the URL directly